spring-integration-azure/src/test/java/com/microsoft/azure/spring/integration/storage/queue/outbound/StorageQueueMessageHandlerTest.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.spring.integration.storage.queue.outbound;
+
+import com.microsoft.azure.spring.integration.core.PartitionSupplier;
+import com.microsoft.azure.spring.integration.storage.queue.StorageQueueOperation;
+import com.microsoft.azure.storage.queue.CloudQueueMessage;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import org.springframework.messaging.Message;
+import org.springframework.messaging.support.GenericMessage;
+import java.util.concurrent.CompletableFuture;
+import static org.mockito.ArgumentMatchers.isA;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+public class StorageQueueMessageHandlerTest {
```

Comment:
I'm unsure you could extend `MessageHanderTest`. But if you want to implement your  own, you should cover as many cases as other.